dart/lib/src/client.dart

```diff
@@ -44,20 +43,14 @@ abstract class SentryClient {
       return emptyFuture;
     }
 
-    event = _applyScope(event: event, scope: scope);
+    event = scope?.applyToEvent(event, hint) ?? event;
```

Comment:
I prefer to log if `scope` is null, so avoid `scope?` here, also, scope processing could return a null event because of event processors, so ` ?? event` would still keep the event non-null which is wrong

Reply:
✅